steam_wrapper/steam_wrapper.py

```diff
@@ -181,9 +149,12 @@ def migrate_data():
     """
     source = os.path.expandvars("$XDG_DATA_HOME")
     target = DATA
+    backup = f'{DATA}.bak'
     steam_home = os.path.join(source, "Steam")
     xdg_data_home = os.path.join(STEAM_ROOT, target)
     if not os.path.islink(source):
+        if os.path.isdir(target):
+            copytree(target, backup)
```

Comment:
We should either exclude `.local/share/Steam` here or do not backup data at all. Otherwise, we can end up backing up the whole Steam library.

Reply:
There's now code handling this scenario.